cli/pkg/kctrl/cmd/app/app.go

```diff
@@ -29,7 +28,38 @@ func isOwnedByPackageInstall(app *kcv1alpha1.App) bool {
 	return false
 }
 
-func getAppDescription(name string, namespace string) string {
-	description := fmt.Sprintf("app/%s (kappctrl.k14s.io/v1alpha1)  namespace: %s", name, namespace)
-	return description
+func appStatusString(app *kcv1alpha1.App) string {
```

Comment:
one thing to consider if this function should return ValueFmt directly. that way you dont have to calculate Error bool externally. alternatively you can also return (string, bool) and use ValueFmt.

Reply:
I think returning (string, bool) makes more sense (in case this ends up being used someplace else.

PS: Me and praveen have been meaning to create an issue to move stuff common to Pkgi and app like waiting logic, and status checks to be scoped to respective objects, with the primary intention of deducing. I guess keeping it generic and not returning a ValueFmt object makes more sense in that scenario 🤔

Reply:
Changed up the function to return (string, bool)
And the bool is used to determine failure status